src/lib/utils/styling/browser-provider.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {
+  APP_BOOTSTRAP_LISTENER,
+  PLATFORM_ID,
+  InjectionToken,    // tslint:disable-line:no-unused-variable
+  ComponentRef,      // tslint:disable-line:no-unused-variable
```

Comment:
Why do these unused symbols need to stay here?

Reply:
Because otherwise we get the following TS error: 

```
error TS4023: Exported variable 'MATCH_MEDIA_PROVIDER' has or is using name 'InjectionToken' from external module "/Users/main/flex-layout/node_modules/@angular/core/src/di/injection_token" but cannot be named
```

Reply:
You can avoid the suppression by explicitly using them in the code where they are now inferred.